remove remote services, the code is not necessary to use this project as a nebula driver

peczenyj opened the pull request:
Hello

this patch should fix issue #199

as you can see, each deleted file is a specific program with a main function.

it is not necessary for this project IMHO

Best Regards

Tiago

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vesoft-inc/nebula-go?pullRequest=200) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vesoft-inc/nebula-go?pullRequest=200) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vesoft-inc/nebula-go?pullRequest=200) it.</sub>

Aiee commented:
Hello @peczenyj, you are right that these files are not necessary to be there. Since the parent directories of these files contain only 1 file, maybe you can delete the directory as well to make it cleaner.

wey-gu commented:
<img width="131" alt="Screen Shot 2022-05-05 at 3 40 20 PM" src="https://user-images.githubusercontent.com/1651790/166880770-472905bb-78e9-4043-9347-1e93fff8a73a.png">
I love this....

Thanks a ton, @peczenyj !

peczenyj commented:
> Hello @peczenyj, you are right that these files are not necessary to be there. Since the parent directories of these files contain only 1 file, maybe you can delete the directory as well to make it cleaner.

Cleaned and Rebased

Aiee reviewed (approved):
LGTM. Good job!
